src/lib/default-project/project-data.js

```diff
@@ -0,0 +1,102 @@
+import ScratchBlocks from 'scratch-blocks';
+
```

Comment:
The alternative approach would be to convert to .jsx like https://github.com/LLK/scratch-gui/blob/develop/src/lib/libraries/decks/index.jsx, and use `FormattedMessage`. I think the ScratchBlocks messages made sense for dynamic menus, but it's not as clear for the default assets.

Reply:
I think the default project requiring `scratch-blocks` is kind of an odd dependency. Won't this also make pulling out the strings more complicated than if it followed the same process as the rest of the GUI?

Reply:
So I'm mistaken, this can't use the jsx approach used by the libraries because the json will end up with `<FormattedMessage>` React components as data. That's fine for the libraries because they're used by the Library component. But project data is used by the VM, so it won't work with react components.